QoL: diable mod when the mod directory starts with "-"

kskudlik opened the pull request:
Just quickly did the QoL thing I mentioned in #12 
I think that's a nice to have when tinkering with different mods

Don´t know if you like how I did it tho. I´m still at a beginner level.

kskudlik commented:
I also added another example that removes the bad language filter, which annoyed many people because it censored them in single player and also removed some unnecessary stuff.

For example, the game itself usees a ship name that is censored.

xforce commented:
Thank you for your work, I know I didn't add a style guide yet, but generally I use PascalCase for class and function names and snake_case for variables.

I would like code to be as simple as possible, so the IsModEnabled function
can be simplified using std::filesystem to just be this:
```
static bool IsModEnabled(fs::path path)
{
    // If mod folder name starts with '-', we don't enable it.
    return path.stem().wstring().find(L'-') != 0;
}
```

kskudlik commented:
well.. that's a heck of a low simpler : D

- added the now working dds files
- changed IsModEnabled to as you mentioned
- tested it everything is working fine
